### ISSUE TITLE
help about btc getTransactionsHistory issue

### ISSUE THREAD

**Issue by mousehaohao** [essentiaone/essentia-bridges-api-ios#28]
Hi Team,
    When i use the api getTransactionsHistory, it always return "unknownError", and I found the log says that ""Essentia⚠️ The data couldn’t be read because it is missing."  But when I visit the url directly in browser, it returns the correct data. any suggestion to fix this?
   The code is quite simple as belows
   ```
        bitcoinWallet.getTransactionsHistory(for: self._address, result: { result in
            switch result {
            case let .success(txHistory):
                //let txs: BitcoinTransactionsHistory.
                ewLogger.debug("[BTC][TxHistory]\(txHistory)")
            case let.failure(error):
                ewLogger.error("[BTC] get history error, \(error)")
            }})
```

**Comment by impul** [comment]
Hello @mousehaohao.
Please provide your BTC address

**Comment by mousehaohao** [comment]
Hi, @impul , here's the address: 12nsfDX1tmvnqHPBQJo564iSLFoqJVNDnS

**Comment by impul** [comment]
Fixed at #29  (0.4.6)
